myst_nbs/time_series/Air_passengers-Prophet_with_Bayesian_workflow.myst.md

```diff
@@ -321,21 +354,26 @@ Neat!
 
 ## Conclusion
 
-We saw how we could implement a Prophet-like model ourselves and fit it to the air passengers dataset. Prophet is an awesome library and a net-positive to the community, but by implementing it ourselves, however, we can take whichever components of it we think are relevant to our problem, customise them, and carry out the Bayesian workflow. Next time you have a time series problem, I hope you will try implementing your own probabilistic model rather than using Prophet as a "black-box" whose arguments are tuneable hyperparameters.
+We saw how we could implement a Prophet-like model ourselves and fit it to the air passengers dataset. Prophet is an awesome library and a net-positive to the community, but by implementing it ourselves, however, we can take whichever components of it we think are relevant to our problem, customise them, and carry out the Bayesian workflow {cite:p}`gelman2020bayesian`). Next time you have a time series problem, I hope you will try implementing your own probabilistic model rather than using Prophet as a "black-box" whose arguments are tuneable hyperparameters.
 
 For reference, you might also want to check out:
 - [TimeSeeers](https://github.com/MBrouns/timeseers), a hierarchical Bayesian Time Series model based on Facebooks Prophet, written in PyMC3
-- [pmprophet](https://github.com/luke14free/pm-prophet), Pymc3-based universal time series prediction and decomposition library inspired by Facebook Prophet
+- [PM-Prophet](https://github.com/luke14free/pm-prophet), a Pymc3-based universal time series prediction and decomposition library inspired by Facebook Prophet
 
-----
++++
 
-Author: [Marco Gorelli](https://github.com/MarcoGorelli)
+## Authors
+* Authored by [Marco Gorelli](https://github.com/MarcoGorelli) in June, 2021 ([pymc-examples#183](https://github.com/pymc-devs/pymc-examples/pull/183))
+* Re-executed by Danh Phan in May, 2022 ([pymc-examples#320](https://github.com/pymc-devs/pymc-examples/pull/320))
```

Comment:
I would say you updated the notebook, and also add youself to the metadata so you appear at the citation suggestion at the bottom.